README.rst

```diff
@@ -669,6 +669,9 @@ of all new files. If your secrets are stored under a specific directory, like a
 ``git`` repository, you can create a ``.sops.yaml`` configuration file at the root
 directory to define which keys are used for which filename.
 
+Note: The file needs to be named ``.sops.yaml``. Other names (i.e. ``.sops.yml``) won't be automatically
+discovered by sops. You'll need to pass the ``--config .sops.yml`` option for it to be picked up.
```

Comment:
```suggestion
.. note::

  The file needs to be named ``.sops.yaml``. Other names (i.e. ``.sops.yml``) won't be automatically
  discovered by SOPS. You'll need to pass the ``--config .sops.yml`` option for it to be picked up.
```

Reply:
I was gonna use rst note admonition syntax, but none of the other notes in the file are using it and also. It doesn't renders nicely in GitHub, it juts adds one extra line break 😢. See [this issue](https://github.com/github/markup/issues/1682) for reference.

Anyways, I'll do the change if you still think is fine. Such a minor thing, I don't want to waste your valuable time.